ImperatorToCk3/Source/Common/Version.h

```diff
@@ -0,0 +1,39 @@
+#ifndef VERSION_H
```

Comment:
Add tests. Double-check that the Imperator version number matches this.

Reply:
I'm not sure if it's resolved, I'd like to add tests for all operators.